crates/miden-lib/asm/note_scripts/BURN.masm

```diff
@@ -0,0 +1,27 @@
+use.miden::contracts::faucets->faucet
```

Comment:
Nit: In other places we don't do this rename, so I'd probably not do it here either unless there's another reason.